scripts/make-package-json-publish-ready.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Define the base directory of your monorepo
+BASE_DIR=$(pwd)
+# Define the directory containing your packages
+PACKAGES_DIR="$BASE_DIR/packages"
+
+# Iterate over each package directory
+for package in "$PACKAGES_DIR"/*; do
+  if [ -d "$package" ]; then
+    PACKAGE_JSON="$package/package.json"
+    if [ -f "$PACKAGE_JSON" ]; then
+      echo "Cleaning $PACKAGE_JSON..."
+      jq 'del(.dependencies, .devDependencies, .peerDependencies, .optionalDependencies, .overrides, .scripts, .browserslist)' "$PACKAGE_JSON" > "$package/package_cleaned.json" && mv "$package/package_cleaned.json" "$PACKAGE_JSON"
```

Comment:
Can you explain this command? what is it doing?

Reply:
We're deleting certain fields in the JSON file.

Reply:
Is it deleting all the dependencies in the package.json?

Reply:
Yes. We don't need anything for the published package.